mod/metrics/set/type_plus_right/metric_value/value.rb

```diff
@@ -8,7 +8,7 @@ def value_card
 end
 
 def value
-  content
+  item_names.join ", "
```

Comment:
We don't have interface for multiline free-text. It's a phrase card. It would replace linebreaks with commas.